spec/models/post_spec.rb

```diff
@@ -1441,17 +1441,12 @@ def post_with_body(body, user = nil)
 
     after { Discourse.redis.flushdb }
 
-    it "should ignore the unique post validator when hiding a post with similar content as a recent post" do
```

Comment:
Unique check validation is a subset of what `PostValidator` does, so we can remove it now that we added a test that checks `PostValidator` isn't run at all.